### ISSUE TITLE
Computer crash when rendering with Blender and ROCM

### ISSUE THREAD

**Issue by DarethW** [ROCm-Developer-Tools/HIP#3088]
Apologies ahead of time if I am doing any of this wrong or not filing the report correctly, I am a novice when it comes to this and only recently got into the world of Linux. Please guide me and ask for any further info you need. I've provided additional info of my system down below.
I am using Kubuntu 22.04 LTS with the latest 5.3 ROCM and HIP drivers from AMD to get Blender to pick up my GPU, which was successful and for awhile it ran flawlessly, but I've seen now that every once in awhile when I render a scene in Blender, it would freeze and crash the computer.
There would be this graphical visual tearing happening on the screen as it turns back on and off consistently and never stops. You cannot do anything in this state at all and would require a reset of the computer.
I took a video of the incident that you can watch here of it happening after another render:
https://drive.google.com/file/d/11iaH1AkEOy8AE-8rCVF0LFIDnzC-YTAJ/view

After each crash, I would see this in the Compositor Settings of the system:
![ytGCi](https://user-images.githubusercontent.com/118379962/202242739-799c0694-3316-4978-a8b5-5c239d99556c.png)

After this happen a few times above, I unchecked "Enable on startup" and did not Re-enabled this, but I think it still crashed again.
I listen to the warning it was giving and tried to upgrade to the latest stable Mesa drivers.
So I upgraded from an older 20 version to this version:
![hGndQ](https://user-images.githubusercontent.com/118379962/202243513-3485e42e-133d-4477-8117-b99746aa8d87.png)

Sadly it still crashed. Here is the info of my system:
![I9UZj](https://user-images.githubusercontent.com/118379962/202244045-c9f945ac-1a8e-4cb2-ae32-52cd48936959.png)

Is there any logs it generates somewhere after each crash that you guys need? I'll be more than happy to upload them here.

**Comment by abhimeda** [comment]
@DarethW Hi, is your issue resolved on the latest HIP? If so can we close this ticket?

**Comment by arrwdodger** [comment]
I still experience a crash on linux 6.8.1, ROCm 6.0.2, RX6900XT, Blender 4.0.

**Comment by mangupta** [comment]
@arrwdodger : ROCm is not supported on 6.8 linux kernel. Please see https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html#supported-operating-systems for the supported kernel version matrix.

**Comment by DarethW** [comment]
@abhimeda hello!  Sorry for the late reply, I did not see the email notification for some reason until much later.  Actually as a matter of fact, when I used the latest HIP it worked flawlessly!  I also did a complete fresh install of the OS from scratch, and start over with the installation of ROCm and HIP in general.
But thank you regardless for replying! 

This problem is definitely "solved" as of right now and have not experienced any severe bugs while rendering in Blender :)
